### PR TITLE
feat: basic HMPB adjudication

### PR DESCRIPTION
**Pull request by eventualbuddha**

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -163,19 +185,45 @@ export default class Store {
...
+        pdf blob,
+        ballot_style_id varchar(255),
+        precinct_id varchar(255),
+        is_test_ballot boolean,
+        layouts_json text
+      )`
     )
     await this.dbRunAsync(
-      'create unique index if not exists hmpb_templates_1 on hmpb_templates (ballot_style_id, precinct_id, is_test_ballot)'
+      `create unique index if not exists hmpb_templates_1 on hmpb_templates (
```
Comment (benadida) [inline src/store.ts:222]: grey-hair index naming convention nit: `hmpb_templates_idx`

File: src/store.ts [hunk 2]
```diff
@@ -351,35 +401,239 @@ export default class Store {
...
-      filename: row.filename,
-      marks: row.marksJSON ? JSON.parse(row.marksJSON) : undefined,
-      cvr: row.cvrJSON ? JSON.parse(row.cvrJSON) : undefined,
+      ballot,
+      marks: overlay,
+      contests,
+    }
+  }
+
+  public async adjudicateBallot(
```
Comment (benadida) [inline src/store.ts:595]: pedantic nit, `saveBallotAdjudication` or some other naming where the name unambiguously indicates a change in what's stored.

File: src/types.ts [hunk 3]
```diff
@@ -40,3 +59,38 @@ export type BallotMetadata = Omit<
...
+  { marginalMarkMin = 0.1, validMarkMin = 0.8 } = {}
+): boolean | undefined {
+  if (mark.score >= validMarkMin) {
+    return true
+  }
+
+  if (mark.score < marginalMarkMin) {
+    return false
+  }
+}
```
Comment (benadida) [inline src/types.ts:83]: implicit `undefined` return... had me confused for a bit.

File: src/store.ts (hunk 2 above)
Comment (eventualbuddha) [reply src/store.ts]: Done.

File: src/store.ts (hunk 1 above)
Comment (eventualbuddha) [reply src/store.ts]: Done

File: src/types.ts (hunk 3 above)
Comment (eventualbuddha) [reply src/types.ts:85]: Done

**Pull request merged**